net/NetUtil.cpp

```diff
@@ -0,0 +1,88 @@
+/* -*- Mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4; fill-column: 100 -*- */
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <config.h>
+
+#include "NetUtil.hpp"
+
+#include "Socket.hpp"
+#if ENABLE_SSL && !MOBILEAPP
+#include "SslSocket.hpp"
+#endif
+
+#include <netdb.h>
+
+namespace net
+{
+std::shared_ptr<StreamSocket>
+connect(const std::string& host, const std::string& port, const bool encrypted,
```

Comment:
Would be great not to rename the "isSSL" to "encrypted" if possible - "encrypted" is too general in my view, making the code reading a bit harder (at least for me) :-)  Can we keep "isSSL" (or "useSSL" if that does not fit) please?

Reply:
Yes, no good reason to change it. I typed `encrypted` separately, but now changed to isSSL.